qiskit/quantum_info/operators/symplectic/sparse_pauli_op.py

```diff
@@ -404,6 +404,46 @@ def simplify(self, atol=None, rtol=None):
             x = self.paulis.x[non_zero_indexes]
             z = self.paulis.z[non_zero_indexes]
             coeffs = coeffs[non_zero]
+
+        return SparsePauliOp(
+            PauliList.from_symplectic(z, x), coeffs, ignore_pauli_phase=True, copy=False
+        )
+
+    def chop(self, tol=1e-14):
+        """Remove real and imaginary parts of the coefficients that are close to 0.
```

Comment:
Should this say set to zero (maybe thats implied by remove) parts that are less than (or equal) in magnitude to `tol` rather than close to 0. The latter is ambiguous and presumably I could chop it at any value I care to supply though normal usage would be a small value!

Reply:
Adjusted 👍🏻